server/chat/backend/agent/tools/mcp_tools.py

```
@@ -66,6 +66,113 @@ def is_destructive_mcp_tool(tool_name: str) -> bool:
+    if not mcp_result:
+        return None
+    try:
+        content_items = mcp_result.get("content") or []
+        for item in content_items:
+            if isinstance(item, dict) and "text" in item:
+                import json as _json
+                data = _json.loads(item["text"])
+                if isinstance(data, dict) and "size" in data:
+                    return int(data["size"])
+    except Exception:
+        pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)